packages/editor-ui/src/components/AIAssistantChat/AIAssistantChat.vue

```diff
@@ -0,0 +1,225 @@
+<script setup lang="ts">
+import { useI18n } from '@/composables/useI18n';
+import { useUsersStore } from '@/stores/users.store';
+import ChatComponent from '@n8n/chat/components/Chat.vue';
+import { ChatOptionsSymbol, ChatSymbol } from '@n8n/chat/constants';
+import type { Chat, ChatMessage, ChatOptions } from '@n8n/chat/types';
+import type { Ref } from 'vue';
+import { computed, provide, ref } from 'vue';
+import QuickReplies from './QuickReplies.vue';
+import { DateTime } from 'luxon';
+import { useAIStore } from '@/stores/ai.store';
+import { chatEventBus } from '@n8n/chat/event-buses';
+import { onMounted } from 'vue';
+import {
+	AI_ASSISTANT_EXPERIMENT_URLS,
+	AI_ASSISTANT_LOCAL_STORAGE_KEY,
+	MODAL_CONFIRM,
+} from '@/constants';
+import { useStorage } from '@/composables/useStorage';
+import { useMessage } from '@/composables/useMessage';
+import { useTelemetry } from '@/composables/useTelemetry';
+
+const locale = useI18n();
+const telemetry = useTelemetry();
+const { confirm } = useMessage();
+
+const usersStore = useUsersStore();
+const aiStore = useAIStore();
+
+const messages: Ref<ChatMessage[]> = ref([]);
+const waitingForResponse = ref(false);
+const currentSessionId = ref<string>(String(Date.now()));
+const disableChat = ref(false);
+
+const userName = computed(() => usersStore.currentUser?.firstName ?? 'there');
+const latestConnectionInfo = computed(() => aiStore.latestConnectionInfo);
+
+const chatTitle = locale.baseText('aiAssistantChat.title');
+const thanksResponses: ChatMessage[] = [
+	{
+		id: String(DateTime.now().toMillis()),
+		sender: 'bot',
+		text: locale.baseText('aiAssistantChat.response.message1'),
+		createdAt: new Date().toISOString(),
+	},
+	{
+		id: String(DateTime.now().toMillis()),
+		sender: 'bot',
+		text: locale.baseText('aiAssistantChat.response.message2'),
+		createdAt: new Date().toISOString(),
+	},
+	{
+		id: String(DateTime.now().toMillis()),
+		sender: 'bot',
+		text: '🙏',
+		createdAt: new Date().toISOString(),
+	},
+	{
+		id: String(DateTime.now().toMillis()),
+		type: 'component',
+		key: 'QuickReplies',
+		sender: 'user',
+		createdAt: new Date().toISOString(),
```

Comment:
Nit: This would read better if we'd define the date "ids" as consts:
```suggestion
const nowMilliseconds = () => String(DateTime.now().toMillis());
const nowIsoString = () => new Date().toISOString();
const thanksResponses: ChatMessage[] = [
	{
		id: nowMilliseconds(),
		sender: 'bot',
		text: locale.baseText('aiAssistantChat.response.message1'),
		createdAt: nowIsoString(),
	},
	{
		id: nowMilliseconds(),
		sender: 'bot',
		text: locale.baseText('aiAssistantChat.response.message2'),
		createdAt: nowIsoString(),
	},
	{
		id: nowMilliseconds(),
		sender: 'bot',
		text: '🙏',
		createdAt: nowIsoString(),
	},
	{
		id: nowMilliseconds(),
		type: 'component',
		key: 'QuickReplies',
		sender: 'user',
		createdAt: nowIsoString(),
```

Also for messages bellow

Reply:
Agree. Will update.